__tests__/app.test.js

```diff
@@ -283,6 +283,125 @@ describe('GET /api/articles/:article_id/comments', () => {
       });
   });
 });
+describe('POST /api/articles', () => {
+
+  const newArticle = {
+    author: 'lurker',
+    title: 'We need more food',
+    body: 'Despite their aloof demeanor, our feline friends have a compelling way of reminding us that when it comes to food, there\'s simply never enough; a constant meow-chorus underscoring the eternal truth - cats believe in a world where more is always better.',
+    topic: 'cats',
+    article_img_url: 'https://images.unsplash.com/photo-1599572739984-8ae9388f23b5?q=80&w=3870&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D',
+  };
+
+  test('POST:201 should insert new article to DB and return it', () => {
+    return request(app).post('/api/articles/').send(newArticle)
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.article).toBeInstanceOf(Object);
+        expect(body.article).toHaveProperty('author');
+        expect(body.article).toHaveProperty('title');
+        expect(body.article).toHaveProperty('body');
+        expect(body.article).toHaveProperty('topic');
+        expect(body.article).toHaveProperty('article_img_url');
+        expect(body.article).toHaveProperty('article_id');
+        expect(body.article).toHaveProperty('votes');
+        expect(body.article).toHaveProperty('created_at');
+        expect(body.article).toHaveProperty('comment_count');
+        expect(typeof body.article.article_id).toBe('number');
+        expect(typeof body.article.votes).toBe('number');
+        expect(typeof body.article.created_at).toBe('string');
+        expect(typeof body.article.comment_count).toBe('string');
+
+        // GET:200 sends the new article to the client
+        return request(app).get('/api/articles/14').expect(200).then(({ body }) => {
```

Comment:
I dont think we need to test it like this, if it comes back with a `created_at` and an `article_id` we can be sure enough that it has been processed correctly